Celeste_Launcher_Gui/Services/GameService.cs

```diff
@@ -273,7 +273,14 @@ public static async Task StartGame(bool isOffline = false)
                         $"--email \"{CurrentEmail}\" --password \"{CurrentPassword.GetValue()}\" --online-ip \"{LegacyBootstrapper.UserConfig.MpSettings.PublicIp}\" --ignore_rest LauncherLang={lang} LauncherLocale=1033";
 
                 Logger.Information("Starting game {@GameExecutable} at {@GamePath}", spartanPath, gamePath);
-                Process.Start(new ProcessStartInfo(spartanPath, arg) { WorkingDirectory = gamePath });
+                Process SProcess = Process.Start(new ProcessStartInfo(spartanPath, arg) { WorkingDirectory = gamePath });
```

Comment:
```suggestion
                var gameProcess = Process.Start(new ProcessStartInfo(spartanPath, arg) { WorkingDirectory = gamePath });
```
We normally use `var` instead of explicitly refering to the type, and variable names usually start with lowercase